docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single node.md

```diff
@@ -0,0 +1,761 @@
+
+For this tutorial we need to create two accounts
+1. The host - the host will load the smart contract
+2. The challenger - the challenger will play the game with the host.
+
+#### Commands
+To use the command line to create the accounts run the following commands. Make sure that the local wallet is [open](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/wallet/open) and [unlocked](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/wallet/unlock) and run these commands to create the accounts with the private keys stored in the local wallet.
+
+Create a wallet called 'local'.
+
+```shell
+cleos wallet create --name local --to-console
+```
+
+This command outputs to stdout
+```console
+Creating wallet: local
+Save password to use in the future to unlock this wallet.
+Without password imported keys will not be retrievable.
+"PW5K8dtLxB8q56JbT8UA5FQyDykNT24LS6i6gcBuxbYSixFgEpFa7"
+```
+
+Save the password, you will need this to unlock the wallet later.
+
+To check the wallet run
+```shell 
+cleos wallet list
+```
+
+The output shows two wallets, one for each version of this tutorial, and * shows that the local wallet is unlocked.
+```console
+Wallets:
+[
+  "local *",
+  "testnet"
+]
+```
+Create two key pairs, one for each account.
+
+```shell
+cleos create key --to-console
+```
+```console
+Private key: 5JSRUrUVbRsV2yJ2XSMtRtPzQ5UKbSYEGEjdKfGMS1xsvRZj7FH
+Public key: EOS5p55prHwrN6KosqF4NdRayVW2mqwA8RGNEbaZXRBs2SQHwBWSf
+```
+```shell
+cleos create key --to-console
+```
+```console
+Private key: 5JReVMTiiztAUyQGp9w7BMMm1HVUurDmEKuSL53DQww3JKVZjot
+Public key: EOS7qkiVnptc8wbHzHPC9jj1YECKJgQeUktBTm8RDA64oH3e75QW5
+```
+
+Create the host account.  
+```shell
+cleos create account eosio host EOS5p55prHwrN6KosqF4NdRayVW2mqwA8RGNEbaZXRBs2SQHwBWSf
+```
+
+Import the matching private key to the local wallet
+```shell
+cleos wallet import --name local --private-key 5JSRUrUVbRsV2yJ2XSMtRtPzQ5UKbSYEGEjdKfGMS1xsvRZj7FH
+```
+
+Create the challenger account and import a private key into the local wallet. 
+```shell
+cleos create account eosio challenger EOS7qkiVnptc8wbHzHPC9jj1YECKJgQeUktBTm8RDA64oH3e75QW5
+```
+
+Import the matching private key to the local wallet
+```shell
+cleos wallet import --name local --private-key 5JReVMTiiztAUyQGp9w7BMMm1HVUurDmEKuSL53DQww3JKVZjot
+```
+
+[[note | Note]]
+| You can also use three or more accounts, one for the smart contract and separate accounts for the host(s) and the challenger(s).
+
+[[warning | Keep your keys safe]]
+| Use a wallet to securely store private keys. Keep your private keys private and do not share your private keys with anyone. A private key provides full access to a blockchain account.
+
+## Tic-tac-toe Smart Contract
+Once you have your eosio accounts you can create the smart contract. Use your favorite text editor of IDE to create the .hpp and.cpp files.
+
+### Smart Contract Requirements
```

Comment:
Throughout the sections below, please make sure to use single codeblocks when referring to filenames, C++ constructs, folder names, etc. (**except** when used in URLs/links):
"tictactoe.hpp" --> "\`tictactoe.hpp\`"
"std::vector" --> "\`std::vector\`"
etc...
It really looks more appealing and easier to the eye when reading.